tests/model/test_grids.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from cascade.model.grids import PriorGrid, AgeTimeGrid, _any_close
+from cascade.model.priors import GaussianPrior
+
+
+@pytest.fixture
+def age_time_grid():
+    return AgeTimeGrid.uniform(age_start=0, age_end=120, age_step=5, time_start=1990, time_end=2018, time_step=1)
+
+
+# def test_PriorGrid__development_target():
+#    # Make a grid using a standard structure. Other options might be
```

Comment:
These commented-out parts. Should they be comments or code or gone?

Reply:
Crap, no that's a mistake. I wanted to leave that in but I also wanted to see coverage with it gone so that I remembered to write explicit tests for each of the pieces. It hides coverage by exercising everything without individually testing anything.